content/docs/next-release/atm.md

```diff
@@ -0,0 +1,304 @@
+    style PROMETHEUS_EXPORTER fill:#404ca8,color:white
+{{< /mermaid >}}
+
+## Metrics Created
+
+Though more in scope of the [OpenTelemetry Collector][opentelemetry-collector],
+it is worth understanding the additional metrics and time series that the
+[SpanMetrics Processor][spanmetrics] will generate in metrics storage to help
+with capacity planning when deploying ATM.
+
+Please refer to [Prometheus documentation][prom-metric-labels] covering the
+concepts of metric names, types, labels and time series; terms that will be used
+in the remainder of this section.
+
+Two metric names will be created:
+- `calls_total`
+  - **Type**: counter
+  - **Description**: counts the total number of spans, including error spans.
+    Call counts are differentiated from errors via the `status_code` label. Errors
+    are identified as any time series with the label `status_code = "STATUS_CODE_ERROR"`.
+- `latency`
+  - **Type**: histogram
+  - **Description**: a histogram of span latencies. Under the hood, Prometheus histograms
+    will create a number of time series:
+    - `latency_count`: The total number of data points across all buckets in the histogram.
+    - `latency_sum`: The sum of all data point values.
+    - `latency_bucket`: A collection of `n` time series (where `n` is the number of
+      latency buckets) for each latency bucket identified by an `le` (less than
+      or equal to) label. The `latency_bucket` counter with lowest `le` and
+      `le >= span latency` will be incremented for each span.
+
+The following formula aims to provide some guidance on the number new time series created:
+```
+num_status_codes * num_span_kinds * (num_calls_metrics + num_latency_buckets) * num_operations
+
+Where:
+  num_status_codes = 3 max (typically 2: ok/error)
+  num_span_kinds = 6 max (typically 2: client/server)
+  num_calls_metrics = 1
+  num_latency_buckets = 17 default
+```
+
+Plugging those numbers in, assuming default configuration (no custom dimensions/labels
+or latency buckets):
+```
+max = 324 * num_operations
+typical = 72 * num_operations
+```
+
+## Configuration
+
+### Enabling ATM
+
+The following configuration is required to enable the ATM feature:
+
+- [Jaeger UI](../frontend-ui#monitor-experimental)
+- [Jaeger Query](../cli#jaeger-query-prometheus)
+  - Set the `METRICS_STORAGE_TYPE` environment variable to `prometheus`.
+  - Optional: Set `--prometheus.server-url` (or `PROMETHEUS_SERVER_URL` environment variable)
+    to the URL of the prometheus server. Default: http://localhost:9090.
+
+## API
+
+Used by the Monitor tab of Jaeger UI to populate the metrics for its visualizations.
+
+### gRPC/Protobuf
+
+The recommended way to programmatically retrieve traces and other data is via `jaeger.api_v2.metrics.MetricsQueryService` gRPC endpoint defined in the [metricsquery.proto][metricsquery.proto] IDL file.
+
+### HTTP JSON
+
+#### R.E.D. Metrics Queries
+
+`/api/metrics/{metric_type}?{query}`
+
+Where (in Backus-Naur form):
```

Comment:
I realize you put a lot of work into BNF here, but I don't think it's the most readable spec for a REST API, usually those are presented in much friendlier format like tables of arguments. One option we could use is to create a swagger definition for the API and use standard tools to generate the docs for it.

Separate question though - why do we even need the spec for this API in the doc, since it's all internal? Maybe it can be just in the README in the repo (or even just as a swagger file)

Reply:
Yes, I would remove it from the docs. Can leave a link to the README from the APIs page.

Reply:
actually APIs page links back to here, so just point to README for details of the API

Reply:
Addressed in [85c4967](https://github.com/jaegertracing/documentation/pull/556/commits/85c49679af4064315432c1f55ced713a12cbd41a).